caql/CaQLLexer.g4

```diff
@@ -0,0 +1,107 @@
+lexer grammar CaQLLexer;
+
+// All keywords in PromQL are case insensitive, it is just function,
+// label and metric names that are not.

```

Comment:
Does it make sense to use [`caseInsensitive`](https://github.com/antlr/antlr4/blob/master/doc/options.md#caseinsensitive) option for this grammar?

Reply:
our business is migrating caql to Grafana(Prometheus QL), so we need caseInsensitive.But only for parsing caql , it is not necessary.I will remove this comment.